### PR TITLE
RIA-3992 update accessibility tests for IA - ExpertUI pages

### PR DESCRIPTION
**Pull request by mounikahmcts**
https://tools.hmcts.net/jira/browse/RIA-3992
https://tools.hmcts.net/jira/browse/TA-397
Accessibility for IAC expertui pages.

https://build.platform.hmcts.net/job/HMCTS_Nightly_IAC/job/ia-ccd-e2e-tests/view/change-requests/job/PR-406/29/artifact/functional-output/xui/a11y.html

### CODE REVIEW

File: e2e/ia.conf.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module.exports = {
 
   CcdGatewayUrl: process.env.TEST_E2E_URL_GATEWAY || 'https://gateway-ccd.aat.platform.hmcts.net',
-  CcdWebUrl: process.env.TEST_E2E_URL_WEB || 'https://manage-case.aat.platform.hmcts.net',
+  CcdWebUrl: 'https://manage-case.aat.platform.hmcts.net',
```
Comment (rohith-tiru) [inline e2e/ia.conf.js:4]: Can you please revert back this change

File: reporter/Report.html [hunk 2]
```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
```
Comment (rohith-tiru) [inline reporter/Report.html:1]: I don't think we need to check in Report.html. This can be removed from the PR

Comment (mounikahmcts) [reply reporter/Report.html:1]: This is a custom report i am generating for accessibility so i have to pass this html template for report generation

**Review by makowalski — DISMISSED** [review]
LGTM

**Review by rohith-tiru — APPROVED** [review]

**Pull request merged**